psi4/driver/qcdb/align.py

```diff
@@ -0,0 +1,737 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
```

Comment:
It might be good to try and clear these out. Py27 should have be pretty transparent.

Reply:
I'm not following. Won't we have lots of tuples printing in py27 and need to add a lot of `1.` to ensure float division if we clear these out?